src/crispy/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ set(crispy_SOURCES
 add_library(crispy-core ${crispy_SOURCES})
 add_library(crispy::core ALIAS crispy-core)
 
-set(CRISPY_CORE_LIBS range-v3::range-v3 fmt::fmt-header-only unicode::core GSL)
+set(CRISPY_CORE_LIBS range-v3::range-v3 fmt::fmt-header-only unicode::core Microsoft.GSL::GSL)
```

Comment:
And this works for all platforms? I hope CI will give us a :+1: here :)

Reply:
As far as I can see, this is the way how you're supposed to do it: https://github.com/Microsoft/GSL#usage-in-cmake

It built on all the platforms that the CI tests for :man_shrugging: I guess it is fine then.